rskj-core/src/test/java/co/rsk/peg/federation/constants/FederationConstantsTest.java

```diff
@@ -130,22 +131,26 @@ private static Stream<Arguments> fedActivationAgeAndActivationArgs() {
         long fedActivationAgeLegacyMainnet = 18500L;
         long fedActivationAgeLegacyTestnet = 60L;
         long fedActivationAgeLegacyRegtest = 10L;
-        ActivationConfig.ForBlock activationsPreRSKIP383 = mock(ActivationConfig.ForBlock.class);
-        when(activationsPreRSKIP383.isActive(ConsensusRule.RSKIP383)).thenReturn(false);
+        ActivationConfig.ForBlock activationsPreRSKIP383 = ActivationConfigsForTest.hop401().forBlock(0L);
 
         long fedActivationAgeMainnet = 40320L;
         long fedActivationAgeTestnet = 120L;
         long fedActivationAgeRegtest = 150L;
-        ActivationConfig.ForBlock activationsPostRSKIP383 = mock(ActivationConfig.ForBlock.class);
-        when(activationsPostRSKIP383.isActive(ConsensusRule.RSKIP383)).thenReturn(true);
+        ActivationConfig.ForBlock activationsPostRSKIP383PreRSKIP419 = ActivationConfigsForTest.fingerroot500().forBlock(0L);
```

Comment:
```suggestion
        ActivationConfig.ForBlock fingerroot500Activations = ActivationConfigsForTest.fingerroot500().forBlock(0L);
```

Let's just use the name of the hard fork, wdyt?